codebuild_specs/run_e2e_tests_linux.yml

```diff
@@ -26,19 +27,18 @@ phases:
   build:
     commands:
       # you can provide a codebuild source version to use old cache and skip all other jobs :)
+      # - export CODEBUILD_SOURCE_VERSION=00f31ea25fc27aa865198398e5014ea4d6a126d9
       - export NODE_OPTIONS=--max-old-space-size=8096
-      # - export CODEBUILD_SOURCE_VERSION=4610160556171f344780aeb2f0d01b7b20147b5d
       - export AMPLIFY_DIR=$CODEBUILD_SRC_DIR/out
       - export AMPLIFY_PATH=$CODEBUILD_SRC_DIR/out/amplify-pkg-linux-x64
       - echo $AMPLIFY_DIR
       - echo $AMPLIFY_PATH
       - source ./shared-scripts.sh && _runE2ETestsLinux
-      - codebuild-breakpoint
   post_build:
     commands:
       - aws sts get-caller-identity
       - source ./shared-scripts.sh && _scanArtifacts
-
 artifacts:
   files:
-    - $CODEBUILD_SRC_DIR/packages/amplify-e2e-tests/amplify-e2e-reports/*
+    - '**/*'
```

Comment:
Why are you uploading artifacts like this instead of specifying specific files?

Reply:
Specific file names are unknown at the time of upload. Since all of these files are named with a GUID we have to specify the base directory (E2E_TEST_COVERAGE_DIR) and take all files in that directory. It should start clean at the beginning of each run and only have coverage files in it after a run is complete.